rc/ansi.kak

```diff
@@ -67,7 +67,11 @@ define-command \
     set-option buffer ansi_color_ranges %val{timestamp}
 }
 
-hook -group ansi global BufCreate '\*stdin(?:-\d+)?\*' %{
+define-command \
+    -docstring %{ansi-enable: render buffer and enable rendering new fifo data for current buffer.} \
+    -params 0 \
```

Comment:
I think these `-params 0` lines should be removed because it's the default, but that doesn't have to be in this PR.

Reply:
I actually like the explicitness.